app/views/users/settings/teams/show.html.erb

```diff
@@ -10,10 +10,14 @@
     <%= render partial: "users/settings/teams/breadcrumbs.html.erb" %>
 
     <!-- TITLE -->
-    <%= link_to team_name_path(@team, format: :json),
-                remote: true,
-                class: 'name-link name-refresh',
-                style: 'color: inherit' do %>
+    <% if can_update_project?(@project) %>
```

Comment:
What is this doing here? Shouldn't this be `can_update_team?`?

Reply:
Yup, that's what you get for hurrying up...

Reply:
Done.